### PR TITLE
Fix plan-patches behavior

### PR DESCRIPTION
**Pull request by abg**
plan-patches currently does not work.  This appears to be because the default working directory of the resource command is `/root` and not the actual source directory.    The plan patches are evaluated relative to the working directory and fail.   The out command is swallowing errors when applying patches, so plan-patches are effectively ignored.

This PR does two things:

- Configures the out command's working directory to the source directory
- Checks the error from `ApplyPlanPatches` and fails if patches cannot be copied

### CODE REVIEW

**Comment by ramonskie** [comment]
it looks good to me.
but we currently cannot test it.
as the gcp storage/archiving is currently broken. (working on that though)

**Comment by ramonskie** [comment]
acceptance tests work when rebasing on master.

File: acceptance/out_test.go [hunk 1]
```diff
@@ -182,6 +194,58 @@ var _ = Describe("out", func() {
...
+			upOutInput   *bytes.Buffer
+			tmpdir       string
+		)
+
+		BeforeEach(func() {
+			var err error
+			tmpdir, err = ioutil.TempDir(os.TempDir(), "")
+			Expect(err).NotTo(HaveOccurred())
+
+			name = fmt.Sprintf("bsr-test-out-plan-%d-%s", GinkgoParallelNode(), projectId)
```
Comment (ramonskie) [inline acceptance/out_test.go:210]: we need to change to `GinkgoParallelProcess` as we moved to ginkgo 2

File: acceptance/out_test.go [hunk 2]
```diff
@@ -113,6 +114,10 @@ var _ = Describe("out", func() {
 		)
 
 		BeforeEach(func() {
+			var err error
+			tmpdir, err = ioutil.TempDir(os.TempDir(), "")
+			Expect(err).NotTo(HaveOccurred())
+
 			name = fmt.Sprintf("bsr-test-out-plan-%d-%s", GinkgoParallelNode(), projectId)
```
Comment (ramonskie) [inline acceptance/out_test.go]: we need to change to `GinkgoParallelProcess` as we moved to ginkgo 2

**Review by rkoster — APPROVED** [review]

**Comment by rkoster** [comment]
Thanks! @abg

**Pull request merged**